src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -140,17 +134,133 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
 
         internal RpcWorkerConfig Config => _workerConfig;
 
+        private void ProcessItem(InboundGrpcEvent msg)
+        {
+            // note this method is a thread-pool (QueueUserWorkItem) entry-point
+            try
+            {
+                switch (msg.MessageType)
+                {
+                    case MsgType.RpcLog when msg.Message.RpcLog.LogCategory == RpcLogCategory.System:
+                        SystemLog(msg);
+                        break;
+                    case MsgType.RpcLog:
+                        Log(msg);
+                        break;
+                    case MsgType.WorkerStatusResponse:
+                        ReceiveWorkerStatusResponse(msg.Message.RequestId, msg.Message.WorkerStatusResponse);
+                        break;
+                    case MsgType.InvocationResponse:
+                        _ = InvokeResponse(msg.Message.InvocationResponse);
+                        break;
+                    default:
+                        ProcessRegisteredGrpcCallbacks(msg);
+                        break;
+                }
+            }
+            catch
+            {
+                // TODO: log?
```

Comment:
Yeah, let's add a log here so we can trace this if we get CRIs

Reply:
@liliankasem Can you please add the logging intended here? I'm not sure what the expectations are, probably quicker to push change to branch directly

Reply:
Done! #8604 8604